crates/uv-toolchain/src/managed.rs

```diff
@@ -283,6 +287,19 @@ impl InstalledToolchain {
             ToolchainRequest::Version(version) => version.matches_version(&self.python_version),
         }
     }
+
+    /// Ensure the toolchain is marked as externally managed with the
+    /// standard `EXTERNALLY-MANAGED` file.
+    pub fn ensure_externally_managed(&self) -> Result<(), Error> {
+        let file = self
+            .path
+            .join("install")
+            .join("lib")
+            .join(format!("python{}", self.python_version.python_version()))
```

Comment:
Is this correct on all platforms?

Reply:
Good question.. umm... I guess I'll need to extract a Windows one and check!

Reply:
I think technically you need to query the interpreter for this.

Reply:
Technically, yep! That's how I found out where to put it. But

- This struct doesn't have access to an interpreter right now
- It seems expensive to query Python to check for this file
- This is just for the managed toolchains where I think we can assume a consistent structure

Reply:
That makes sense. If it's just for the managed Pythons, we may be able to make some assumptions. (Won't we need to query Python eventually any way though?)

Reply:
We do but not until you "use" the toolchain — in `uv toolchain install` we never actually need to query the interpreter. Idk how fast the operation really needs to be since it'll be dominated by the time it takes to download but it's not like we're already doing it.

Reply:
The Windows builds use `Lib` so this should be generally fine? I guess I'll capitalize it.